DURATION: 30 # seconds
CAN_INTERFACE : "vcan0"
CAN_HZ : 1000000 # Hz

#lighting
EVENT_FRAME:
    CAN_ID : 1
    SIGNAL:
        START_BIT: 0
        LENGTH: 8
        SCALE: 1.0
        OFFSET: 0.0
        VALUE: 1.0

